### DISCUSSION TITLE
Unit tests disabled for default profile. Use contributor profile to run unit tests.

### DISCUSSION THREAD

**Discussion by dsgrieve** [microsoft/gctoolkit#48] [Announcements]
By default, unit tests are disabled. This configuration avoids the download and unpacking of assets from microsoft/gctoolkit-testdata, which are rather large files. 

To run the unit tests, use `-Pcontributor` to enable the contributor profile. See [CONTRIBUTING](https://github.com/microsoft/gctoolkit/blob/main/CONTRIBUTING.md) for details. 

Thanks to @anantk24 for this contribution!

**Comment by anantk24** [comment]
@dsgrieve I think this is not true  now, we might need to modify contributing markdown .
